x-pack/plugins/security_solution/server/usage/detections/detections_helpers.ts

```diff
@@ -172,7 +172,7 @@ export const getMlJobsUsage = async (ml: MlPluginSetup | undefined): Promise<MlJ
 
   if (ml) {
     try {
-      const fakeRequest = {} as KibanaRequest;
+      const fakeRequest = { headers: {} } as KibanaRequest;
```

Comment:
Does this part need to be fixed as well?
https://github.com/elastic/kibana/blob/master/x-pack/plugins/security_solution/server/lib/detection_engine/signals/signal_rule_alert_type.ts#L212

Reply:
Good question! For now, it's only the `modulesProvider` that performs the additional headers check, the `jobsProvider` (and thus ML Rule execution) continues to function as expected.